RELEASE.md

```diff
@@ -41,8 +41,7 @@
 
 ## Deprecations
 
-
-*   KFP v1 SDK 
+*   KFP v1 SDK
```

Comment:
To provide further context, let's include:

* KubeflowDagRunner (KFP v1 SDK) is deprecated. Use KubeflowV2DagRunner (KFP v2 pipeline spec) instead.

Reply:
sure @nikelite, Updated